downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md

```diff
@@ -59,6 +59,10 @@ Annotations are not allowed on the tuple rest descriptor.
 
 ### Bug fixes
 
+#### Field annotation access in record type descriptors without a type definition
+
+Field annotation values of record type descriptors defined without a type definition can now be accessed at runtime.
```

Comment:
```suggestion
- Annotation values of fields of record type descriptors that are not defined with a type definition are now accessible at runtime
```

Reply:
Not addressed completely? We don't need headings under bug fixes.

Reply:
My bad, I feel that only need to update the description